### PR TITLE
Update ReadMe.md

### PR DESCRIPTION
**Pull request by francis2054**
Added instructions for new WIP flashing method using the Marauder app on the Flipper Zero

### CODE REVIEW

File: ReadMe.md [hunk 1]
```diff
@@ -10,15 +10,28 @@ Requires a connected dev board running Marauder FW. [See install instructions fr
...
+4. Now for transferring the files to the Flipper Zero, drag all the files from the folder on your desktop to the "Marauder" folder inside "apps_data" folder on the Flipper Zero SD card. Preferred method to transfer these files is plugging the SD card into your computer with an adapter, but qFlipper works as well. Insert the Flipper Zero SD Card back into the Flipper before proceeding to the next step.
+5. Plug your Wi-Fi Devboard into the Flipper.
+6. Press and keep holding the boot button while you press the reset button once, release the boot button after 2 seconds.
+7. Open the Marauder app on your Flipper Zero, it should be named "esp32_wifi_marauder" and be located under Apps->GPIO from the main menu if you followed the instructions for how to install the app further up on this page. (You might get an API mismatch error if the Flipper firmware you are running doesn't match the files you've downloaded, you can try "Continue" anyway, otherwise the app needs to be rebuilt or you might need to update the firmware on your Flipper.
```
Comment (0xchocolate) [inline ReadMe.md]: Add a closing paren here )

File: ReadMe.md [hunk 2]
```diff
@@ -10,15 +10,28 @@ Requires a connected dev board running Marauder FW. [See install instructions fr
    GitHub action workflow, and select the most recent run, scroll down to artifacts.
 3. The FAP is built for the `dev` and `release` channels of both official and unleashed
    firmware. Download the artifact corresponding to your firmware version.
-4. Extract `ESP32_WiFi_Marauder.fap` from the ZIP file to your Flipper
-   Zero SD card, preferably under Apps/GPIO along with the rest of the GPIO apps.
+4. (Optional step to avoid confusion) Go to "Apps/GPIO" on the Flipper SD Card, delete any existing Marauder app, on some firmwares there will be a `ESP32CAM_Marauder.fap` or similar.
+5. Extract `esp32_wifi_marauder.fap` from the ZIP file downloaded in step 3 to your Flipper Zero SD card, preferably under Apps/GPIO along with the rest of the GPIO apps. (If you're using qFlipper to transfer files you need to extract the content of the ZIP file to your computer before you drag it to qFlipper, as qFlipper does not support direct dragging from a ZIP file (at least on Windows).
```
Comment (0xchocolate) [inline ReadMe.md]: Add a closing paren ) at the end here

**Review by 0xchocolate — COMMENTED** [review]
Can you also add your name to the readme? Like maybe right under "Get the app", something like "Documentation updated by @francis2054 ". (Optional but might help.)

**Review by francis2054 — COMMENTED** [review]
Added guide for flashing with WIP method for official Wifi Devboard

File: ReadMe.md [hunk 3]
```diff
@@ -10,15 +10,28 @@ Requires a connected dev board running Marauder FW. [See install instructions fr
...
 
 From a local clone of this repo, you can also build the app yourself using ufbt.
 
 ## In-app ESP32 flasher (WIP)
```
Comment (0xchocolate) [inline ReadMe.md:18]: Actually might make more sense to add your name and comment under this heading. (Your call :) )

**Review by francis2054 — COMMENTED** [review]
Small edits and added credit for guide

**Review by francis2054 — COMMENTED** [review]
Edited credit

**Pull request merged**